TFT/src/User/includes.h

```diff
@@ -91,7 +91,7 @@
 #include "ScreenShot.h"
 #include "SerialConnection.h"
 #include "Settings.h"
-#include "SmartFeatures.h"
+#include "AddonHardware.h"
```

Comment:
should be better to put it on alfabetic order.
Also isn't it better to call invert_axis as inverted_axis?

Reply:
invert_axis is ok. No reason to increase the name length when it already gives the correct meaning.

Reply:
I mean also the meaning of the data. With "invert" do you mean the verb "to invert"? The data structures should not report a verb. The verb is used in function names

Reply:
With invert_axis here means an instruction or order. Here it is still a variable and is used as an array of bits. If I had swapped the words, then it can be like axis_inverted.

Reply:
yes, so my interpretation was correct (it is an action or order). I would try to not use an order for the data naming, so I would use axis_inverted (although I thought that the more appropriate english syntax was inverted_axis). I have already made some cleanup on my next PR. E.g. swapping Emulate M600 to Emulated M600. When possible, all the conf. params should not provide an order/action in the naming (it's not used in italian language but I think also in english language).

Reply:
invert_axis, inverted_axis, axis_inverted all mean the same thing. I would not bother about renaming the variables as long as the names reflect their purpose. Renaming existing variables also makes it difficult to track changes across a long range of commits.